build/signing-config-mac.yaml

```diff
@@ -0,0 +1,48 @@
+# This file describes the code signing configuration for macOS.
+
+# List of entitlements.
+entitlements:
+  # This contains the default entitlements, for files not otherwise listed.
+  default:
+  - com.apple.security.inherit
+
+  # Entitlement overrides.  This is a list of overrides, each with a "paths"
+  # key describing which paths to override, and an "entitlements" key for the
+  # overriding entitlements.
+  overrides:
+  - paths:
+    - '' # This is the main application
+    entitlements:
+    - com.apple.security.cs.allow-jit
+  - paths:
+    - Contents/Resources/resources/darwin/lima/bin/limactl
+    - Contents/Resources/resources/darwin/lima/bin/limactl.ventura
+    - Contents/Resources/resources/darwin/lima/bin/qemu-system-aarch64
+    entitlements:
+    - com.apple.security.hypervisor
+    - com.apple.security.virtualization
```

Comment:
I think qemu doesn't need `virtualization`, only limactl does. However, limactl should also get the `network.server` and `network.client` entitlements.

Reply:
Just realized that only `limactl.ventura` does need `virtualization`, `network.server`, and `network.client`. The legacy `limactl` binary is only used on platforms that don't support VZ emulation.